### PR TITLE
Add sample pages for Lightning design system

### PR DESCRIPTION
**Pull request by hyork**
These sample pages were built by Clark Cregg and they're excellent reference guide or starter pages for users of Lightning DS in Skuid.

### CODE REVIEW

File: Design_Systems/Lightning/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+A matching design system for Skuid builders who create apps to match Salesforce's [Lightning Design System](https://www.lightningdesignsystem.com/). This design system is one of the default design systems in Skuid, so you don't have to import it. You can create a cloned version to make customizations.
+
+In this folder, you'll find a few sample pages that closely match the more common tabs in Salesforce like List view and Detail view. You can use these sample pages as a reference guide or starter pages.
+
+## Screenshots
+### List View
+<img src="Screenshots/listview.png" alt="Screenshot: List View" width="600"/>
+
+### Detail View
+<img src="Screenshots/detailview.png" alt="Screenshot: Detail View" width="600"/>
```
Comment (Rnhatch) [inline Design_Systems/Lightning/README.md:13]: @hyork  Can you do two things here: 
1. Add an instructions block
2. Add links - especially back to these pages in the demo org for internal users. 

Look at this page for a model you can copy. 
https://github.com/skuid/SamplePages/tree/master/Use_Cases/Directory

File: Design_Systems/Lightning/DetailView.xml [hunk 2]
```diff
@@ -0,0 +1,338 @@
...
+																					</fields>
+																				</section>
+																			</sections>
+																		</column>
+																	</columns>
+																	<styles>
+																		<spacing top="1.5" bottom="1.5" left="1.5" right="1.5"/>
+																	</styles>
+																</skuid__form>
+															</components>
```
Comment (Rnhatch) [inline Design_Systems/Lightning/DetailView.xml:259]: Can we get a save / cancel button with the form.  Just so the page is slightly functional?

**Comment by Rnhatch** [comment]
General philosophical question.  These pages are "design only".   None of the functions are wired up.  No buttons work,  Search does not search,  you can't save...    But maybe that's ok.  We should either do some work to add minimal function,  or add a pretty clear note in the ReadMe that nothing works and this is visual exploration only.

**Comment by itmightbecody** [comment]
Wow, these are great for showing how closely the updated design system matches Lightning. So good!

Since this is mostly intended to show off that visual aesthetic, I'm personally okay with not having the functionality wired up. That said, if we do that, we need **big bold letters at the top that the Readme saying these pages aren't functional and are intended for visual demonstration only.**

File: Design_Systems/Lightning/DetailView.xml (hunk 2 above)
Comment (hyork) [reply Design_Systems/Lightning/DetailView.xml:259]: If we note that the page isn't functional, do you still want the save/cancel buttons added and wired up?

Comment (Rnhatch) [reply Design_Systems/Lightning/DetailView.xml:259]: No - we can do without them.  I'll resolve this conversation.

**Review by Rnhatch — APPROVED** [review]
This all looks good.

**Pull request merged**